ntp-proto/src/clock.rs

```diff
@@ -196,17 +203,31 @@ impl<C: NtpClock> ClockController<C> {
             self.poll_interval_counter -= self.preferred_poll_interval.as_log() as i32;
         }
 
+        trace!(
+            counter = debug(self.poll_interval_counter),
+            "Poll preference"
+        );
+
         // If our preference becomes strong enough, adjust poll interval
         // and reset. The hysteresis here ensures we aren't constantly flip-flopping
         // between different preferred interval lengths.
         if self.poll_interval_counter > Self::POLL_ADJUST {
             self.poll_interval_counter = 0;
             self.preferred_poll_interval = self.preferred_poll_interval.inc();
+            debug!(
+                poll_interval = debug(self.preferred_poll_interval),
```

Comment:
You can use `?self.preferred_poll_interval`, which is a good deal more concise than wrapping `debug()` around it.